hook!-full-game/src/Player/States/Wall.gd

```diff
@@ -0,0 +1,39 @@
+extends State
+
+const WALL_SLIDE_SPEED: = 100.0
+const WALL_PULL_STRENGTH: = 300.0
+
+var _jump_strength: = 1000.0
+var _velocity: = Vector2.ZERO
+var _wall_direction: = -1
```

Comment:
Is viable holds the opposite of the actual wall direction. Should be either `wall_normal_direction` or the value should be * -1